pkg/authentication/scram/mock_types_test.go

```diff
@@ -0,0 +1,101 @@
+package scram
```

Comment:
this defines a set of mock implementations to break any dependency on the `mdbv1` package.